Makefile

```diff
@@ -157,7 +166,7 @@ login: ## Login to the container. Note: if the container is already running, log
 	fi
 
 .PHONY: gen-cmake
-gen-cmake: ## Generate cmake files, used internally
+gen-cmake: create-ccache-folder ## Generate cmake files, used internally
```

Comment:
If the cache directory is under bft, you can use mkdir to create it instead of a separate target

Reply:
cach folder (as we discussed) should not be under bft